cvat/apps/dataset_manager/project.py

```diff
@@ -160,7 +160,7 @@ def data(self) -> dict:
         raise NotImplementedError()
 
 @transaction.atomic
-def import_dataset_as_project(project_id, dataset_file, format_name, conv_mask_to_poly):
+def import_dataset_as_project(src_file, project_id, format_name, conv_mask_to_poly):
```

Comment:
What is the point of this change?

Reply:
Because all import functions (`import_dataset_as_project`, `import_task_annotations`, `import_job_annotations`, 
 including backups import `_import_project`, `_import_task`) will be called from another function `import_resource_with_clean_up_after` that will remove the uploaded file.